hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/InfoBucketHandler.java

```diff
@@ -40,7 +41,56 @@ public void execute(OzoneClient client, OzoneAddress address)
         .getVolume(address.getVolumeName())
         .getBucket(address.getBucketName());
 
-    printObjectAsJson(bucket);
+    if (bucket.getSourceBucket() != null && bucket.getSourceVolume() != null) {
```

Comment:
nit. Do we need to check both bucket and volume != null? Is bucket check alone good enough?

Reply:
Creation of link does not allow if any of them is empty, technically one check should be okay, but it would be better to check both as safe check to be clear I have thought.

Let me know if you still want it only one needs to be checked.